content/200-orm/200-prisma-client/000-setup-and-configuration/010-generating-prisma-client.mdx

```diff
@@ -4,34 +4,43 @@ metaTitle: 'Generating Prisma Client'
 metaDescription: 'This page explains how to generate Prisma Client. It also provides additional context on the generated client, typical workflows and Node.js configuration.'
 ---
 
-<TopBlock>
+Prisma Client is a generated database client that's tailored to your database schema. By default, Prisma Client is generated into the `node_modules/.prisma/client` folder, but we highly recommend [you specify an output location](#using-a-custom-output-path).
 
-Prisma Client is an auto-generated database client that's tailored to your database schema. By default, Prisma Client is generated into the `node_modules/.prisma/client` folder, but [you can specify a custom location](#using-a-custom-output-path).
+:::warning
+
+In Prisma ORM 7, Prisma Client will no longer be generated in `node_modules` by default and will require an output path to be defined. [Learn more below on how to define an output path](#using-a-custom-output-path)
```

Comment:
can we maybe link this to the AMA? mentioning these changes about an upcoming major version change without any additional context may be too confusing for readers.